src/index.html

```diff
@@ -183,6 +183,121 @@ <h2 class="intro__heading">Dream. Develop. Deploy!</h2>
       </div>
     </div>
   </section>
+
+  <section class="FAQ faq__wrapper" id="FAQ">
+    <!--img class="faq-image-left" src="static/faq-symbols-left.svg" />
+    <img class="faq-image-right" src="static/faq-symbols-right.svg"><OPTION 2 UNCCOMMENT THIS TO TAKE A LOOK -->
+    <div class="container">
+      <h2>Frequently Asked Questions</h2>
+      <div class="flex-container">
+        <div class="left-container">
+          <div class="accordion">
+            <div class="background">
+              <div class="accordion-item">
+                <a>Who can participate?</a>
+                <div class="content">
+                  <p>All undergraduate and high school students are welcome! If you are under 18, we’ll need a parental consent form.</p>
+                </div>
+              </div>
+            </div>
+            <br>
+            <div class="background">
+              <div class="accordion-item">
+                <a>What should I bring?</a>
+                <div class="content">
+                  <p>Your student ID, laptop, phone, chargers, some sleeping equipment like pillows and sleeping bags (we’ll be providing air mattresses).</p>
+                </div>
+              </div>
+            </div>      
+            <br>
+            <div class="background">
+              <div class="accordion-item">
+                <a>When should I arrive?</a>
+                <div class="content">
+                  <p>Registration will take place at the St-James Theatre (265 St Jacques St, Montreal, QC H2Y 1M6). Out-of-town check-in begins at 9:00 am, and local check-in begins at 9:30 am.</p>
```

Comment:
Can we make `St-James Theatre (265 St Jacques St, Montreal, QC H2Y 1M6)` link to a google maps page?